t/performance_probability.t

```diff
@@ -0,0 +1,78 @@
+#Performance Probability test.
+use 5.010;
+use strict;
+use warnings;
+
+use Date::Utility;
```

Comment:
Could you, please, convert the `cvs` file to let it uses epochs, to get rid of `D::U`?

`D::U` uses Moose, which depends on 100+ modules on CPAN. It would be not very people-friendly or CO2-friendly to force install Moose just to test probablity :)